src/disjoint_set.jl

```diff
@@ -109,15 +143,15 @@ end
     intmap::Dict{T,Int}
     internal::IntDisjointSets
 
-    function (::Type{DisjointSets{T}}){T}(xs)    # xs must be iterable
+    function (::Type{DisjointSets{T}}){T}(xs, store_size::Bool=false)    # xs must be iterable
```

Comment:
I believe that `Val` is often discouraged, but I'm wondering if this would be a valid use case?